README.md

```diff
@@ -33,19 +33,9 @@ If you want to keep the local documentation alive after closing the prompt, you
 
 ## License
 
-The Telerik® UI for Blazor Documentation is licensed under an MIT license. This license applies to the markdown (.md) files in this site **ONLY**, and does not convey, override or modify any existing licenses covering the runtime source and components of Telerik® UI for Blazor. For information about available licenses for the Telerik® UI for Blazor click [here](https://www.telerik.com/purchase/license-agreement/blazor-ui).
+The **Telerik® UI for Blazor Documentation** is licensed under the **Apache 2.0** license. See the [`LICENSE` file](LICENSE) for the full terms. This license applies to the markdown (`.md`) files in this site **ONLY**, and does not convey, override or modify any existing licenses covering the runtime source and components of Telerik® UI for Blazor. For information about [available licenses for the Telerik® UI for Blazor click here](https://www.telerik.com/purchase/license-agreement/blazor-ui).
```

Comment:
* Don't use bold for emphasis. The first sentence is brief and clear enough so you don't need to put more weight on the words in bold. The Style Guide allows [sparse and careful use of italics](https://docs.telerik.com/style-guide/element-formatting) when introducing new terms and emphasis is needed.
* Use [descriptive anchor text](https://docs.telerik.com/style-guide/cross-references#guidelines) instead of "click here" etc.
```suggestion
The Telerik® UI for Blazor Documentation is licensed under the Apache 2.0 license. See the [`LICENSE` file](LICENSE) for the full terms. This license applies to the markdown (`.md`) files in this site ONLY and does not convey, override, or modify any existing licenses covering the runtime source and components of Telerik® UI for Blazor. For more information, see the [available Telerik® UI for Blazor license agreement](https://www.telerik.com/purchase/license-agreement/blazor-ui).
```